DependencyInjection/Configuration.php

```diff
@@ -47,6 +47,7 @@ public function getConfigTreeBuilder()
                     'If profiler is disabled the tracer service will be disabled as well.'
                 )
             ->end()
+            ->scalarNode('app_root_class')->defaultValue('App\\Kernel')->end()
```

Comment:
maybe we should set this parameter only when there exists no `App\AppBundle` class. 

```suggestion
            ->scalarNode('app_root_class')->defaultValue(class_exists('App\\AppBundle') ? null : 'App\\Kernel')->end()
```